File: Aulas/Programação Script/Outros/Aula 01/Exemplos/Choque de realidade/main.py
import time

def processamento_pesado():
    print("🐍 Python: Iniciando contagem...")
    start = time.time()

    soma = 0
    for i in range(100_000_000):
        soma += i

    end = time.time()
    print(f"Resultado: {soma}")
    print(f"Tempo: {end - start:.4f} segundos")

if __name__ == "__main__":
    processamento_pesado()

File: Aulas/Programação Script/Outros/Aula 01/Exemplos/Choque de realidade/main.go
package main

import (
	"fmt"
	"time"
)

func main() {
	fmt.Println("🐹 Go: Iniciando contagem...")
	start := time.Now()

	var soma int64 = 0
	for i := 0; i < 100_000_000; i++ {
		soma += int64(i)
	}

	duration := time.Since(start)
	fmt.Printf("Resultado: %d\n", soma)
	fmt.Printf("Tempo: %f segundos\n", duration.Seconds())
}
